docs/tracing_reference.rst

```diff
@@ -147,6 +147,20 @@ TraceConfig
 
       ``params`` is :class:`aiohttp.TraceRequestStartParams` instance.
 
+   .. attribute:: on_request_chunk_sent
```

Comment:
Please also upgrade flow diagrams at the beginning of file.

Reply:
done. Here I added new signals to description. Please let me know if you meant something more elaborate.
![zrzut ekranu z 2018-02-27 15-04-09](https://user-images.githubusercontent.com/287481/36732980-8e915330-1bcf-11e8-843f-66ddbd6c7413.png)